test/integration/command_test.exs

```diff
@@ -110,7 +110,22 @@ defmodule Integration.CommandTest do
   end
 
   test "running a command in a pipeline with nil output", %{user: user} do
-    response = send_message(user, ~s(@bot: seed '[{"a": "1", "b": "2"}, {"a": "3"}]' | filter --field="b" | echo $a))
+    response = send_message(user, ~s(@bot: seed '[{"a": "1", "b": "2"}, {"a": "3"}]' | filter --path=b | echo $a))
     assert response["data"]["response"] == "1"
   end
+
+  test "reading the path to filter a certain path", %{user: user} do
+    response = send_message(user, ~s(@bot: seed '[{"foo":{"bar":{"baz":"stuff"}}}, {"foo": {"bar":{"baz":"me"}}}]' | operable:filter --path="foo.bar"))
+    assert response["data"]["response"] == "{\n  \"bar\": {\n    \"baz\": \"stuff\"\n  }\n}\n{\n  \"bar\": {\n    \"baz\": \"me\"\n  }\n}"
```

Comment:
Also, some code I was experimenting with on an unpushed branch might come in useful for dealing with the JSON strings we get back:

```elixir
# Currently, it's impossible to get a JSON array of data out of the
  # end of a pipeline. Instead, we get newline-separated JSON strings
  # of each individual data map in that array
  #
  # Example:
  #     "{\n  \"foo\": 123\n}\n{\n  \"foo\": 321\n}"
  #
  # This function recovers an Elixir data structure from this string,
  # by relying on the fact that the boundary between maps will be
  # "}\n{", which is done with a regex with positive lookahead and
  # lookbehind.
  #
  # Now you have `n` problems.
  defp unmangle_multiple_output(output_string),
    do: output_string |> String.split(~r/(?<=})\n(?={)/) |> Enum.map(&Poison.decode!(&1, keys: :atoms!))
```

Might be worthwhile to add that into a test helper module and take advantage of it here, at least until we change how we test commands in #248